vita3k/renderer/include/renderer/vulkan/screen_filters.h

```diff
@@ -41,6 +41,7 @@ class ScreenFilter {
     virtual bool need_post_processing_render_pass() {
         return false;
     }
+    virtual ~ScreenFilter() = default;
```

Comment:
It's an old known bug. https://github.com/Vita3K/Vita3K/pull/2915 
Recheck it. Maybe now this fix is working

Reply:
I checked it and after fixing the assertion I did not see any errors in the log.

Reply:
So you can change Screen filter between FXAA and FSR when game is running? There was crash.

Reply:
I don't have FSR in the dropdown, but switch filters works without a crash.

Reply:
I checked on latest build on this branch, and got crash. 
How to reproduce:
* Start any game.
* Go to Settings/Gpu and change Screen filter to anything else
* Press Save & Apply.

Last log lines before actual crash
```

[21:52:33.934] |E| [debug_callback]: Validation layer: Validation Error: [ UNASSIGNED-CoreValidation-DrawState-InvalidCommandBuffer-VkPipeline ] Object 0: handle = 0x23290417be0, type = VK_OBJECT_TYPE_COMMAND_BUFFER; Object 1: handle = 0x314c0a0000000746, type = VK_OBJECT_TYPE_PIPELINE; | MessageID = 0xea7fd65c | You are adding vkCmdEndRenderPass() to VkCommandBuffer 0x23290417be0[] that is invalid because bound VkPipeline 0x314c0a0000000746[] was destroyed.
[21:52:33.934] |E| [debug_callback]: Validation layer: Validation Error: [ UNASSIGNED-CoreValidation-DrawState-InvalidCommandBuffer-VkDescriptorSet ] Object 0: handle = 0x23290417be0, type = VK_OBJECT_TYPE_COMMAND_BUFFER; Object 1: handle = 0xaeec13000000073f, type = VK_OBJECT_TYPE_DESCRIPTOR_SET; | MessageID = 0xe8616bf2 | You are adding vkCmdEndRenderPass() to VkCommandBuffer 0x23290417be0[] that is invalid because bound VkDescriptorSet 0xaeec13000000073f[] was destroyed or updated.
[21:52:33.935] |E| [debug_callback]: Validation layer: Validation Error: [ UNASSIGNED-CoreValidation-DrawState-InvalidCommandBuffer-VkDescriptorPool ] Object 0: handle = 0x23290417be0, type = VK_OBJECT_TYPE_COMMAND_BUFFER; Object 1: handle = 0xe472c2000000073e, type = VK_OBJECT_TYPE_DESCRIPTOR_POOL; | MessageID = 0x4e1644a0 | You are adding vkCmdEndRenderPass() to VkCommandBuffer 0x23290417be0[] that is invalid because bound VkDescriptorPool 0xe472c2000000073e[] was destroyed.
[21:52:33.936] |E| [debug_callback]: Validation layer: Validation Error: [ UNASSIGNED-CoreValidation-DrawState-InvalidCommandBuffer-VkSampler ] Object 0: handle = 0x23290417be0, type = VK_OBJECT_TYPE_COMMAND_BUFFER; Object 1: handle = 0xfbc55b0000000747, type = VK_OBJECT_TYPE_SAMPLER; | MessageID = 0x71aefee7 | You are adding vkCmdEndRenderPass() to VkCommandBuffer 0x23290417be0[] that is invalid because bound VkSampler 0xfbc55b0000000747[] was destroyed.
[21:52:33.936] |E| [debug_callback]: Validation layer: Validation Error: [ UNASSIGNED-CoreValidation-DrawState-InvalidCommandBuffer-VkBuffer ] Object 0: handle = 0x23290417be0, type = VK_OBJECT_TYPE_COMMAND_BUFFER; Object 1: handle = 0x5784b70000000743, type = VK_OBJECT_TYPE_BUFFER; | MessageID = 0x26ed8608 | You are adding vkEndCommandBuffer() to VkCommandBuffer 0x23290417be0[] that is invalid because bound VkBuffer 0x5784b70000000743[] was destroyed.
[21:52:33.937] |E| [debug_callback]: Validation layer: Validation Error: [ UNASSIGNED-CoreValidation-DrawState-InvalidCommandBuffer-VkPipeline ] Object 0: handle = 0x23290417be0, type = VK_OBJECT_TYPE_COMMAND_BUFFER; Object 1: handle = 0x314c0a0000000746, type = VK_OBJECT_TYPE_PIPELINE; | MessageID = 0xea7fd65c | You are adding vkEndCommandBuffer() to VkCommandBuffer 0x23290417be0[] that is invalid because bound VkPipeline 0x314c0a0000000746[] was destroyed.
[21:52:33.938] |E| [debug_callback]: Validation layer: Validation Error: [ UNASSIGNED-CoreValidation-DrawState-InvalidCommandBuffer-VkDescriptorSet ] Object 0: handle = 0x23290417be0, type = VK_OBJECT_TYPE_COMMAND_BUFFER; Object 1: handle = 0xaeec13000000073f, type = VK_OBJECT_TYPE_DESCRIPTOR_SET; | MessageID = 0xe8616bf2 | You are adding vkEndCommandBuffer() to VkCommandBuffer 0x23290417be0[] that is invalid because bound VkDescriptorSet 0xaeec13000000073f[] was destroyed or updated.
[21:52:33.940] |E| [debug_callback]: Validation layer: Validation Error: [ UNASSIGNED-CoreValidation-DrawState-InvalidCommandBuffer-VkDescriptorPool ] Object 0: handle = 0x23290417be0, type = VK_OBJECT_TYPE_COMMAND_BUFFER; Object 1: handle = 0xe472c2000000073e, type = VK_OBJECT_TYPE_DESCRIPTOR_POOL; | MessageID = 0x4e1644a0 | You are adding vkEndCommandBuffer() to VkCommandBuffer 0x23290417be0[] that is invalid because bound VkDescriptorPool 0xe472c2000000073e[] was destroyed.
[21:52:33.940] |E| [debug_callback]: Validation layer: Validation Error: [ UNASSIGNED-CoreValidation-DrawState-InvalidCommandBuffer-VkSampler ] Object 0: handle = 0x23290417be0, type = VK_OBJECT_TYPE_COMMAND_BUFFER; Object 1: handle = 0xfbc55b0000000747, type = VK_OBJECT_TYPE_SAMPLER; | MessageID = 0x71aefee7 | You are adding vkEndCommandBuffer() to VkCommandBuffer 0x23290417be0[] that is invalid because bound VkSampler 0xfbc55b0000000747[] was destroyed.
[21:52:33.942] |C| [logging::exception_handler]: Exception EXCEPTION_ACCESS_VIOLATION (0xc0000005). 
[21:52:33.942] |C| [logging::exception_handler]: Read violation at address 0x60.
```

Reply:
I still can not reproduce, but I will revert this change.

Reply:
Thank you very much.